evmbin/src/main.rs

```diff
@@ -14,7 +14,21 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Parity EVM interpreter binary.
+//! Parity EVM Interpreter Binary.
+//!
+//! ## Overview
+//!
+//! The Parity EVM interpreter binary is an additional tool in the Parity
+//! Ethereum toolchain. It is an EVM implementation for Parity Ethereum that
+//! may be used to run a standalone version of the EVM interpreter.
+//!
+//! ## Usage
+//!
+//! Build Parity Ethereum from source, then start it with:
+//! ```bash
+//! cd evmbin && cargo build -p evmbin && cd ..
```

Comment:
```suggestion
//! cargo build -p evmbin
```

That should suffice, cding should not be required.

Reply:
Guessing here, but most users of `evmbin` would want to build it with `--release`, so it'd become `cargo build --release -p evmbin`.

Reply:
I've changed it to:
```
//! ```bash
//! cargo build -p evmbin --release
//! ./target/release/parity-evm --help
//! ```
```

Is that ok now @dvdplm @tomusdrw  ?